configure

```diff
@@ -5512,6 +5545,7 @@ EXEPRE := $HOSTEXEPRE
 EXEEXT := $HOSTEXEEXT
 NASM := $NASM
 NASMFLAGS := $NASMFLAGS
+PANDOCFLAGS := $PANDOCFLAGS
```

Comment:
I think you might want to also add `PANDOC := $PANDOC` here, and then use it in Makefile.common instead of assuming there that it is named `pandoc` (ignoring the extension) and it is in the path.

Reply:
Fixed, I think we should still keep PANDOCFLAGS in configure to allow users to overwrite it in the future. A good use case is converting to RTF or using an markdown_github instead of gfm inputs on older versions of pandoc

Reply:
OK. But maybe you should split the output name to a different variable:
```
PANDOCFLAGS="-f gfm -t plain"
PANDOCOUTPUT=README
```

Then in Makefile.common you don't need to hardcode the output name when adding it to `DIST_FILES_DOCS`, which would actually allow in configure to use different flags and a different output name to generate a `ReadMe.rtf` instead of a text `README` on macOS for example.

```
ifdef USE_PANDOC
# Convert README.md to plain text for any platform that might require it
readme: README.md
	$(QUIET_PANDOC)$(PANDOC) $(PANDOCFLAGS) -o $(PANDOCOUTPUT) README.md
endif
```
and
```
DIST_FILES_DOCS:=$(addprefix $(srcdir)/,AUTHORS COPYING COPYING.BSD COPYING.LGPL COPYING.FREEFONT COPYRIGHT NEWS README.md)
ifdef USE_PANDOC
DIST_FILES_DOCS+=$(PANDOCOUTPUT)
endif
```

Reply:
I just rebased and updated the branch to handle that. Instead of --use-pandoc there are now two optional flags that will enable pandoc instead:
--with-pandoc-output
--with-pandoc-format

Passing any of these will enable pandoc and convert the README.md file.